custom/inddex/food.py

```diff
@@ -307,9 +307,16 @@ def _calculate_total_grams(recipe, ingredients):
 
 
 class FoodData:
-    def __init__(self, domain, ucr_rows):
-        self.ucr_rows = ucr_rows
+    def __init__(self, domain, *, datespan, case_owners=None, gap_type=None, recall_status=None):
```

Comment:
I really like how you've used `*` to make `FoodCaseData` params explicit.

Reply:
:snake: :three: :muscle: